File: src/main/java/com/ironhack/lab/models/Employee.java
package com.ironhack.lab.models;

import com.fasterxml.jackson.annotation.JsonIgnore;

import javax.persistence.*;

@Entity
public class Employee {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String name;

    private String phoneNumber;

    private String officeNumber;

    private String position;

    private String manager;

    public Employee() {
    }

    public Employee(String name, String phoneNumber, String officeNumber, String position, String manager) {
        this.name = name;
        this.phoneNumber = phoneNumber;
        this.officeNumber = officeNumber;
        this.position = position;
        this.manager = manager;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public String getOfficeNumber() {
        return officeNumber;
    }

    public void setOfficeNumber(String officeNumber) {
        this.officeNumber = officeNumber;
    }

    public String getPosition() {
        return position;
    }

    public void setPosition(String position) {
        this.position = position;
    }

    public String getManager() {
        return manager;
    }

    public void setManager(String manager) {
        this.manager = manager;
    }
}

File: lab6/src/app/models/employee.ts
export class Employee {

    constructor(private _id: number, private _name : string, private _phoneNumber: string, private _officeNumber: string, private _position: string, private _manager: string){}

    get id(){
        return this._id
    }

    set id(id : number){
        this._id = id
    }

    get name(){
        return this._name
    }

    set name(name : string){
        this._name = name
    }

    get phoneNumber(){
        return this._phoneNumber
    }

    set phoneNumber(phoneNumber : string){
        this._phoneNumber = phoneNumber
    }

    get officeNumber(){
        return this._officeNumber
    }

    set officeNumber(officeNumber : string){
        this._officeNumber = officeNumber
    }

    get position(){
        return this._position
    }

    set position(position : string){
        this._position = position
    }

    get manager(){
        return this._manager
    }

    set manager(manager : string){
        this._manager = manager
    }
}
